monero-outreach-docs/translations/es/archive/getting_started_es.md

```diff
@@ -0,0 +1,197 @@
+Traducción de materiales relacionados con Monero. Si hablan otro idioma, revisen esto.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/erciccione-monero-localization](https://taiga.getmonero.org/project/erciccione-monero-localization/) y #monero-translations
+
+#### Outreach
+Haciendo a Monero ampliamente conocido e interpretado.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/26](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/26)
+
+#### Laboratorio de Investigación de Monero
+Realizando investigación con Monero y software relacionado.
+_¿Dónde empezar?_ [web.getmonero.org/es/resources/research-lab](https://web.getmonero.org/es/resources/research-lab/) y #monero-research-lab
+
+#### Equipo de Respuesta a Malware
+Ayudar a aquellos que han sido expuestos a malware, incluyendo a la minería no deseada en su navegador, minería en sus sistemas y ransomware.
+_¿Dónde empezar?_ [mrw.getmonero.org](https://mrw.getmonero.org/) y Mattermost
+
+#### Sitio Web
+Desarrollo del sitio web “oficial” (getmonero.org) y su mantenimiento.
+_¿Dónde empezar?_ [repo.getmonero.org/monero-project/monero-site](https://repo.getmonero.org/monero-project/monero-site) y #monero-site
+
+#### Hardware
+Realizando un diseño abierto en especificaciones para un monedero hardware compatible con Monero y otros proyectos relacionados.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation]](https://taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation/)
+
+#### Soporte
+Ayudando a los usuarios por una variedad de razones, incuye la configuración de monederos, la conexión a nodos, restauración de cuentas, transferencia de fondos y más.
+_¿Dónde empezar?_ [r/MoneroSupport](https://www.reddit.com/r/MoneroMining/), #monero-support y [monero.stackexchange.com](https://monero.stackexchange.com/)
+
+#### Minería
+Asegurando la cadena de bloques de Monero. Incluye discusiones sobre el lanzamiento de nuevos desarrollos de software y guías útiles para la minería en variedad de configuraciones de software y hardware.  
+
+#### Trading
+Discusiones sobre XMR y temas en general sobre especulación del precio.
+_¿Dónde empezar?_ [r/xmrtrader](https://www.reddit.com/r/xmrtrader/) y #monero-markets
+
+#### GUI
+Mejorando la interfaz gráfica del usuario para el software de Monero.
+_¿Dónde empezar?_ [github.com/monero-project/monero-gui](https://github.com/monero-project/monero-gui), #monero-gui y #monero-dev
+
+#### Proceso de respuesta a vulnerabilidades
+Manejo del proceso de respuesta a vulnerabilidades. Se debe aceptar invitación de parte del equipo de trabajo.
+_¿Dónde comenzar?_ [github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md)
+
+#### RIAT - Instituto para la Economía futura
+Promoviendo proyectos de cryptomonedas, incluyendo a Monero. Instituto basado en Vienna.
```

Comment:
+Promoviendo proyectos de **criptomonedas**, incluyendo a Monero. Instituto basado en Vienna._

Reply:
You're right on this.